compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/AggregatedOpInterfaceImpl.cpp

```diff
@@ -177,7 +242,8 @@ OnlineAttentionOp::decomposeOperation(OpBuilder &b) {
   Value oldAcc = getOutput();
   Value oldMax = getMax();
   Value oldSum = getSum();
-  Type elementType = getQuery().getType().getElementType();
+  Type elementType = getElementTypeOrSelf(getResult(0).getType());
```

Comment:
NIT: Can we do `getOutput().getType()` or `oldAcc.getType()`?

Reply:
Done.